EncountifyAPI/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I think this file should be added to .gitignore

Reply:
Do this to remove file from source control
https://sigalambigha.home.blog/2020/03/11/how-to-refresh-gitignore/